dotCMS/src/test/java/com/dotmarketing/business/cache/provider/h22/H22CacheTest.java

```diff
@@ -36,27 +37,37 @@
 	final int numberOfThreads = 40;
 	final int numberOfGroups = 100;
 	final int maxCharOfObjects = 100;
-
+    final private ExecutorService executor  = Executors.newSingleThreadExecutor();
+    
+    final public H22Cache cache = getCache();
+    
+    final H22Cache getCache() {
+        final File dir = new File("/tmp/h2cachetest");
+        dir.delete();
+        dir.mkdirs();
+        return new H22Cache(dir.getAbsolutePath());
+        
+    }
+    
+    
+    @Before
```

Comment:
Codacy found an issue: [JUnit 4 tests that execute tests should use the @Test annotation, JUnit 5 tests should use @Test, @RepeatedTest, @TestFactory, @TestTemplate or @ParameterizedTest](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5796487)